### PR TITLE
9882 ppr marketing uxa

### PR DESCRIPTION
**Pull request by jordiwes**
- ppr marketing feedback based on https://docs.google.com/document/d/1VbzuSlcYaXcULoG8ydk_CDNzXhGHQ25JWjZSkqEoxnQ/edit

### CODE REVIEW

File: assets/scss/base.scss [hunk 1]
```diff
@@ -158,6 +157,18 @@ ul {
...
+  font-size: 1rem;
+  font-weight: normal;
+  letter-spacing: normal;
+  padding: inherit 28px !important;
+  text-transform: none;
+  .v-icon + span,
+  span + .v-icon {
+    margin-left: 0.25rem;
+  }
+}
```
Comment (kialj876) [inline assets/scss/base.scss:170]: ooh nice -- this will fix one of the ux comments I forgot to fix for mine :)

File: pages/ppr-marketing.vue [hunk 2]
```diff
@@ -5,45 +5,56 @@
...
             <v-col cols="6">
               <SbcSigninButton />
             </v-col>
           </v-row>
-          <v-row>
+          <v-row v-if="!loggedIn">
             <v-col>
               New to BC Registries?
-              <a class="font-weight-bold" href="/"
+              <a class="font-weight-bold" href="https://dev.bcregistry.ca/business/auth/choose-authentication-method"
```
Comment (kialj876) [inline pages/ppr-marketing.vue:28]: you'll need to grab this link from the config: $config._

File: pages/ppr-marketing.vue [hunk 3]
```diff
@@ -5,45 +5,56 @@
...
                 >Create a BC Registries Account</a
               >
             </v-col>
           </v-row>
+          <v-row v-if="loggedIn">
+            <v-col>
+              <v-btn
+                  large
+                  class="button-login font-weight-bold mr-2"
+                  to="https://ppr-ui-dev.apps.silver.devops.gov.bc.ca/ppr"
```
Comment (kialj876) [inline pages/ppr-marketing.vue:38]: again you'll need to get it from the config: $config.pprUrl I think

File: pages/ppr-marketing.vue [hunk 4]
```diff
@@ -299,20 +329,30 @@
...
+    ContactInfo,
   },
-  data() {
-    return {
-      /** Whether the user is logged in. */
-      userProfile: false,
-    }
+  computed: {
+    loggedIn() {
+      console.log(sessionStorage.getItem(SessionStorageKeys.KeyCloakToken))
```
Comment (kialj876) [inline pages/ppr-marketing.vue:350]: remove the log pls

**Review by kialj876 — COMMENTED** [review]
Looks good! Just a couple minor things. You might want to rebase + update the ppr-marketing widths so it matches the header / breadcrumb widths as the screen changes too

**Review by doug-lovett — COMMENTED** [review]
Nothing to add in addition to Kial's comments - ppr marketing links need to use env variable values.

File: pages/ppr-marketing.vue (hunk 4 above)
Comment (jordiwes) [reply pages/ppr-marketing.vue]: done

File: pages/ppr-marketing.vue (hunk 3 above)
Comment (jordiwes) [reply pages/ppr-marketing.vue]: hopefully what I put works... is there a place where I can see what the config is?

File: pages/ppr-marketing.vue (hunk 2 above)
Comment (jordiwes) [reply pages/ppr-marketing.vue]: done, I think

File: pages/ppr-marketing.vue (hunk 3 above)
Comment (kialj876) [reply pages/ppr-marketing.vue]: yeah this looks right to me

File: pages/ppr-marketing.vue (hunk 2 above)
Comment (kialj876) [reply pages/ppr-marketing.vue]: I think you may have a double slash here - can you double check locally?

Comment (jordiwes) [reply pages/ppr-marketing.vue]: there was & it's fixed

**Review by kialj876 — APPROVED** [review]

**Review by doug-lovett — APPROVED** [review]
Looks good.

**Pull request merged**